mlflow/server/js/src/common/utils/Utils.js

```diff
@@ -545,10 +545,7 @@ class Utils {
   }
 
   static getAjaxUrl(relativeUrl) {
-    if (process.env.USE_ABSOLUTE_AJAX_URLS === 'true') {
-      return '/' + relativeUrl;
-    }
-    return relativeUrl;
```

Comment:
Previously, we allowed using relative URLs here to enable use cases like https://github.com/mlflow/mlflow/issues/1120 - what's the motivation for removing support for relative URLs here?

Reply:
Sorry I didn't have this use case in mind, but relative urls don't work with BrowserRouter that's why I removed them, but I can put them back if needed.